Pipfile

```diff
@@ -78,6 +78,7 @@ prettytable = ">=3.0.0"
 pycep-parser = "==0.3.7"
 charset-normalizer = "*"
 pyston_lite_autoload = {version = "==2.3.4.1", markers="python_version == '3.8' and (sys_platform=='linux' or sys_platform=='darwin')"}
+pyston-lite = {version = "==2.3.4.1", markers="python_version == '3.8' and (sys_platform=='linux' or sys_platform=='darwin')"}
```

Comment:
pay attention to add it in setup.py if required

Reply:
yes, would be good